src/Sulu/Bundle/MediaBundle/Tests/Unit/Media/Manager/MediaManagerTest.php

```diff
@@ -548,14 +548,13 @@ public static function provideGetByIds(): iterable
      *     0: string,
      *     1: string,
      *     2: string,
-     *     3: string
      * }>
      */
     public static function provideSpecialCharacterFileName()
     {
         return [
-            ['aäüßa', 'aäüßa', 'aaeuesa', ''],
-            ['aäüßa.mp4', 'aäüßa', 'aaeuesa', '.mp4'],
+            ['aäüßa', 'aaussa', ''],
+            ['aäüßa.mp4', 'aaussa', '.mp4'],
```

Comment:
update to the todo list of our slugger to recheck that, can even be that just the case as we mocked it before and I removed the mock mechanic